### PR TITLE
UserWarning: invalid index of a 0-dim tensor.

### PR DESCRIPTION
**Pull request by astier**
A warning is shown when the nec_run.py is executed:
> nec_agent.py:96: UserWarning: invalid index of a 0-dim tensor. This will be an error in PyTorch 0.5. Use tensor.item() to convert a 0-dim tensor to a Python number action = torch.cat(q_estimates).max(0)[1].data[0]

Fix this by replacing
> action = torch.cat(q_estimates).max(0)[1].data[0]

with
> action = torch.cat(q_estimates).max(0)[1].item()

### CODE REVIEW

**Comment by mjacar** [comment]
See my comment in https://github.com/mjacar/pytorch-nec/issues/5. I don't currently have any plans  to keep this repo up to date with the most recent version of PyTorch, so if you want to do so, I would advise you to fork this.
